tutorials/R/Hyperspectral/Intro-hyperspectral/Select-Pixels-Compare-Spectral-Signatures/Select-Pixels-Compare-Spectral-Signatures.Rmd

```diff
@@ -277,7 +279,7 @@ lower and upper bound of each of those atmopheric absopbtion bands is specified
 in the HDF5 file. Let's read those bands and plot rectangles where the 
 reflectance measurements are obscured by atmospheric absorbtion. 
 
-```{r mask-atmospheric-absorbtion-bands, fig.width=9, fig.height=6}
+```{r mask-atmospheric-absorbtion-bands, fig.width=9, fig.height=6, fig.cap="Plot of spectral signatures for the five different land cover types: Field, Tree, Roof, Soil, and Water.Values falling within the two rectangles from the previous image have been set to NA and ommited from the plot. On the x-axis is wavelength in nanometers and on the y-axis is reflectance values."}
```

Comment:
Space needed between sentences

Reply:
Hmm this tutorial should have the five specctral signatures plotted in this order: full signatures, signatures with box overlaid, and then the signatures with the 'gaps' (NAs in absorption bands). Will you please double check these plots and re-arrange these fig.cap(s) as needed?